tests/resources/generator/SN_to_cAMP/SN_to_cAMP_parameters.csv

```diff
@@ -0,0 +1,49 @@
+variable_name,units,value,data_reference
```

Comment:
this file isn't used that I can see.

Reply:
Removed [07fe5d6](https://github.com/cellml/libcellml/pull/1204/commits/07fe5d6633779fb2c6833df803dd5752139726bc)